src/main/resources/assets/circuit/blockstates/cpu.json

```diff
@@ -1,12 +1,14 @@
 {
   "forge_marker": 1,
   "defaults": {
-    "model": "circuit:cpu",
+    "model": "minecraft:cube_top",
     "textures": {
-      "top": "circuit:blocks/cpu_top_off"
+      "top": "circuit:blocks/cpu_top_off",
+      "side": "circuit:blocks/cpu_side"
     }
   },
   "variants": {
+    "inventory": [{}],
```

Comment:
In my testing, it wasn't required. I'm pretty sure it's because cube_top -> cube -> block, where block has a set of default transforms defined.